apps/heft/src/cli/actions/BuildAction.ts

```diff
@@ -12,6 +12,7 @@ export class BuildAction extends HeftActionBase {
   protected _watchFlag: CommandLineFlagParameter;
   protected _productionFlag: CommandLineFlagParameter;
   protected _liteFlag: CommandLineFlagParameter;
+  protected _firstCompilationEmitCallback: (() => void) | undefined;
```

Comment:
Should this be a tappable event so that plugins (i.e. more than one listener) can wait for it?

Is there a way to `await` for a tappable event?

Reply:
Take a look at the latest updates.